pkg/machine/wsl/wutil/wutil.go

```diff
@@ -50,20 +50,13 @@ func IsWSLInstalled() bool {
 
 func IsWSLStoreVersionInstalled() bool {
 	cmd := SilentExecCmd("wsl", "--version")
-	out, err := cmd.StdoutPipe()
+	cmd.Stdout = nil
```

Comment:
Non-blocking nit: consider using a `bytes.Buffer` for stdout and stderr and write that to `logrus.Debugf`.